src/Ledger/Certs.lagda

```diff
@@ -173,6 +198,16 @@ stake distribution anymore. Genesis delegations and MIR certificates
 have been superceded by the new governance mechanisms, in particular
 the \TreasuryWdrl governance action in case of the MIR certificates.
 
+\subsection{Explicit deposits}
+
+Registration and deregistration of staking credentials are now
+required to explicitly state the deposit that is being paid or
+refunded. This aligns them better with other design decisions such as
+having explicit transaction fees and helps make this information
+visible to light clients and hardware wallets. While not shown in the
```

Comment:
What are "light clients?"  Is this defined somewhere?

Reply:
It's not a formal thing, so we don't need to define it. A light client is some piece of software that interacts with the chain but doesn't run a node, which makes it more convenient but less secure. Think browser plugins or really anything that doesn't take forever to sync.